doc/releases/release-notes-3.3.rst

```diff
@@ -1124,6 +1124,11 @@ Libraries / Subsystems
 HALs
 ****
 
+* STM32F1xx HAL
+
+  * Rename and add new pinctrl definitions for PWM entries, to differentiate
+    PWM inputs and PWM outputs
```

Comment:
This is maybe buried too much deep down into the release note and not explicit enough to be useful.
Device tree section would be a better fit and please provide an explicit example.

Also use past tense.

Reply:
Done